src/alias/azext_alias/alias.py

```diff
@@ -210,14 +215,15 @@ def get_full_alias(self, query):
         """
         if query in self.alias_table.sections():
             return query
+
         return next((section for section in self.alias_table.sections() if section.split()[0] == query), '')
 
     def load_full_command_table(self):
         """
         Perform a full load of the command table to get all the reserved command words.
         """
         load_cmd_tbl_func = self.kwargs.get('load_cmd_tbl_func', None)
-        if load_cmd_tbl_func:
+        if callable(load_cmd_tbl_func):
```

Comment:
This worries me a bit. In what situation this variable is not callable? If that's valid scenario it worths consider changing the variable name.

Reply:
I don't think load_cmd_tbl_func will ever be non-callable. Converted the above to:
```python
...
load_cmd_tbl_func = self.kwargs.get('load_cmd_tbl_func', None)
if load_cmd_tbl_func is not None:
...
```